.editorconfig

```diff
@@ -55,6 +55,11 @@ csharp_style_namespace_declarations = file_scoped
 # Brace settings
 csharp_prefer_braces = true # Prefer curly braces even for one line of code
 
+# Whitespace options
+csharp_style_allow_embedded_statements_on_same_line_experimental = false
+csharp_style_allow_blank_lines_between_consecutive_braces_experimental = false
+csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimental = false
```

Comment:
In general, we should wait until experimental features have been released for real before incorporating them in this repo. There's no reason to rush here.
I'd go further there and say we're in the weeds here; #24055 was about FxCop, not StyleCop, and certainly not teensy whitespace rules.